src/components/common/YourCompany/YourCompany.vue

```diff
@@ -269,7 +281,7 @@
 
     <!-- Folio Information -->
     <template v-if="isPremiumAccount">
-      <v-divider class="mx-4" />
+      <v-divider class="mx-4 my-1" />
```

Comment:
Wow, almost 300 lines of template. In another life, we should create sub-components for various sections in here.

Reply:
It has certainly grown! 
Will look a little closer at improving this though, if not now will certainly keep it in mind as we proceed.